src/sentry/snuba/metrics/query_builder.py

```diff
@@ -152,17 +152,34 @@ def resolve_tags(
                 ],
             )
         elif input_.function == "match":
-            column = input_.parameters[0]
+            first_param = input_.parameters[0]
```

Comment:
is any change to rename `first_param` to something more readable.

Reply:
The thing is that a param can have multiple types and i wouldn't know what other name to use, do you have any suggestions?

Reply:
keep old name `column`?

Reply:
It is not a column anymore, the error shown by Sentry contained a case in which `Function(ifNull)` was passed as first param.

Reply:
in sql fn(column) still a column.

Reply:
same logic in sql, `first_param` confuse a lot.  It's same as `result`, `params` etc.
Column is column it can handle value/ or function which evaluates to value.

Reply:
```
input_first_param
```